### PR TITLE
Rename and update master branch references to main

### PR DESCRIPTION
**Pull request by akchinSTC**
This PR also updates the test matrix:
 - Remove Python 3.6 (EOL)
 - Add Python 3.9

### CODE REVIEW

File: visualize/map-from-coordinates.ipynb [hunk 1]
```diff
@@ -21,7 +21,7 @@
    "source": [
     "# @param data_dir temporal data storage for local execution\n",
     "# @param data_csv path and csv file name\n",
-    "# @param master url of master (default: local mode)"
+    "# @param main url of master (default: local mode)"
```
Comment (ptitzler) [inline visualize/map-from-coordinates.ipynb:24]: Not sure what this means and whether anything needs to be updated. The 'master' variable isn't consumed in this notebook and judging by its default value it does not appear to be a branch reference? @romeokienzler

**Comment by romeokienzler** [comment]
@ptitzler this is Apache Spark (or distributed computing) terminology and not related to git

**Review by ptitzler — DISMISSED** [review]
LGTM! Test failure => https://github.com/elyra-ai/component-library/pull/32

**Review by ptitzler — APPROVED** [review]
LGTM. Also updated test matrix to reflect current Elyra Python support.

**Pull request merged**